.github/workflows/build_db_clean_image.yml

```diff
@@ -0,0 +1,37 @@
+name: build_db_clean_image
+
+on:
+  schedule:
+    - cron:  '0 6 * * 6'
+
+env:
+  AWS_DEFAULT_REGION: us-west-2
+
+jobs:
+  build_db_clean_image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
```suggestion
```

I think you can remove this since the scheduled job will be running on the default branch (i.e., not a pull request).